### PR TITLE
Updating base Ubuntu image for Azure

### PR DESCRIPTION
**Pull request by nzthiago**
@scotmoor @rguthriemsft @duftler PTAL

### CODE REVIEW

**Comment by rguthriemsft** [comment]
LGTM

File: rosco-web/config/rosco.yml [hunk 1]
```diff
@@ -133,12 +133,12 @@ azure:
     baseImages:
     - baseImage:
         id: ubuntu
-        shortDescription: v14.04
-        detailedDescription: Ubuntu Server 14.04.4-LTS
+        shortDescription: v14.05
```
Comment (duftler) [inline rosco-web/config/rosco.yml]: Looks like the minor number is still `04`, so this line should remain the same.

Comment (rguthriemsft) [inline rosco-web/config/rosco.yml]: Understood it looks wrong, but when we checked VM skus using our Cli this was correct.

**Comment by duftler** [comment]
Needs a rebase.

**Pull request closed without merging**